agent_deploy/kubernetes/sysdig-daemonset.yaml

```diff
@@ -33,15 +33,18 @@ spec:
           path: /usr
       hostNetwork: true
       hostPID: true
- #    serviceAccount: sysdigcloud                          #OPTIONAL - OpenShift service account for OpenShift
+ #    serviceAccount: sysdigcloud                           #OPTIONAL - OpenShift service account for OpenShift
       containers:
       - name: sysdig-agent
         image: sysdig/agent
+#        imagePullPolicy: Always                            #OPTIONAL - pull up latest image every deployment 
```

Comment:
I would like to rephrase the comment in: `always pull the latest container image tag`